tests/e2e/pageObjects/browser-page.ts

```diff
@@ -1,692 +1,745 @@
+        await t.click(this.removeElementFromListIconButton);
+        await t.typeText(this.countInput, count);
+        await t.click(this.removeElementFromListButton);
+        await t.click(this.confirmRemoveListElementButton);
+    }
+
+    /**
+     * Remove List element from head
+     * @param count The count if elements for removing
+     */
+    async removeListElementFromHead(count: string): Promise<void> {
+        await t.click(this.removeElementFromListIconButton);
+        //Enter count of the removing elements
+        await t.typeText(this.countInput, count);
+        //Select Remove from head selection
+        await t.click(this.removeElementFromListSelect);
+        await t.click(this.removeFromHeadSelection);
+        //Confirm removing
+        await t.click(this.removeElementFromListButton);
+        await t.click(this.confirmRemoveListElementButton);
+    }
+
+    /**
+     * Add json key with value on the same level of the structure
+     * @param jsonKey The json key name
+     * @param jsonKeyValue The value of the json key
+     */
+    async addJsonKeyOnTheSameLevel(jsonKey: string, jsonKeyValue: any): Promise<void> {
+        await t.click(this.addJsonObjectButton);
+        await t.typeText(this.jsonKeyInput, jsonKey);
+        await t.typeText(this.jsonValueInput, jsonKeyValue);
+        await t.click(this.applyButton);
+    }
+
+    /**
+     * Add json key with value inside the Json structure
+     * @param jsonKey The json key name
+     * @param jsonKeyValue The value of the json key
+     */
+    async addJsonKeyInsideStructure(jsonKey: string, jsonKeyValue: any): Promise<void> {
+        await t.click(this.expandJsonObject);
+        await t.click(this.addJsonFieldButton);
+        await t.typeText(this.jsonKeyInput, jsonKey);
+        await t.typeText(this.jsonValueInput, jsonKeyValue);
+        await t.click(this.applyButton);
+    }
+
+    /**
+     * Add json structure
+     * @param jsonStructure The structure of the json key
+     */
+    async addJsonStructure(jsonStructure: string): Promise<void> {
+        await t.click(this.expandJsonObject);
+        await t.click(this.editJsonObjectButton);
+        await t.pressKey('ctrl+a delete');
+        await t.typeText(this.jsonValueInput, jsonStructure);
+        await t.click(this.applyEditButton);
+    }
+
+    /**
+     * Get Values list of the key
+     * @param element Selector of the element with list
+     */
+    async getValuesListByElement(element): Promise<string[]> {
+        const keyValues = [];
+        const count = await element.count;
+        for (let i = 0; i < count; i++) {
+            keyValues[i] = await element.nth(i).textContent;
+            i++;
+        }
+        return keyValues;
+    }
+
+    /**
+     * Check tree view structure
+     * @folders name of folders for tree view build
+     */
+    async checkTreeViewFoldersStructure(folders: Array<Array<string>>, delimiter: string, commonKeyFolder: boolean): Promise<void> {
```

Comment:
1. You missed adding a description for the two last parameters.
2. The `Array<Array<string>>` couldn't you use with `string[]`?

Reply:
1. Added
2. Changed to string[][] (array with arrays)